terraform/environments/electronic-monitoring-data/dms_main.tf

```
@@ -58,8 +58,14 @@ module "dms_second_task" {
 
   for_each = toset(local.is-production ? [
     "capita_alcohol_monitoring",
-    "g4s_emsys_mvp",
+    "civica_orca",
     "g4s_atrium",
+    "g4s_cap_dw",
+    "g4s_centurion"
```

```suggestion
    "g4s_centurion",
```
you need a comma here